src/components/scenes/FioAddressSettingsScene.js

```diff
@@ -65,12 +91,27 @@ class FioAddressSettingsComponent extends React.Component<Props, LocalState> {
     this.setState({ showTransfer: true })
   }
 
+  onAddBundlesPress = () => {
+    this.setState({ showAddBundles: true })
+  }
+
   cancelOperation = () => {
-    this.setState({ showTransfer: false })
+    this.setState({ showTransfer: false, showAddBundles: false })
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name